synapse/lib/webapp.py

```diff
@@ -102,14 +102,14 @@ def getFooByBar(self, bar):
         wapp.main()
 
     '''
-    def __init__(self, **settings):
+    def __init__(self, app_config={}, srv_config={}):
+
         EventBus.__init__(self)
         s_daemon.DmonConf.__init__(self)
-        tornado.web.Application.__init__(self, **settings)
+        tornado.web.Application.__init__(self, **app_config)
```

Comment:
The following are the available app config settings: http://www.tornadoweb.org/en/stable/web.html#tornado.web.Application.settings